package.json

```diff
@@ -41,10 +41,13 @@
     "watcherGlob": "**"
   },
   "scripts": {
+    "prebuild": "yarn extract:map",
     "build": "if test \"$NODE_ENV\" = \"production\"; then yarn build:prod; else yarn build:dev; fi",
     "build:prod": "NODE_ENV=production parcel build src/index.html",
     "build:dev": "NODE_ENV=development parcel build src/index.html --no-minify",
+    "predev": "yarn extract:map",
     "dev": "parcel watch src/index.html",
+    "extract:map": "node ./scripts/extractMap.js",
```

Comment:
I'm not sure how this script does extract anything, maybe it should be renamed to "export" ?

Reply:
I feel like it extracts colliding layer, creates two dimensional array and exports it. So I'm not sure how to name it properly